.github/workflows/tests.yaml

```diff
@@ -25,11 +27,27 @@ jobs:
             python: "3.10"
           - plone: "5.2-latest"
             python: "3.11"
+          - plone: "5.2-latest"
+            python: "3.12"
+          - plone: "6.0-latest"
+            python: 3.7
           - plone: "6.0-latest"
+            python: "3.12"
+          - plone: "6.1-latest"
             python: 3.7
+          - plone: "6.1-latest"
+            python: 3.8
+          - plone: "6.1-latest"
+            python: 3.9
+          - plone: "6.1-latest"
+            python: "3.10"
+          - plone: "6.1-latest"
+            python: "3.11"
+          - plone: "6.1-latest"
+            python: "3.12"
```

Comment:
@ericof If you're skipping the tests on 3.12, then we shouldn't claim that it is supported.